src/colorizer/ColorizeCanvas.ts

```diff
@@ -389,15 +386,6 @@ export default class ColorizeCanvas implements IRenderCanvas {
     }
   }
 
-  public setBackgroundColor(color: Color): void {
-    this.setUniform("backgroundColor", color);
-  }
-
-  /** Set the color of the area outside the frame in the canvas. */
-  public setCanvasBackgroundColor(color: Color): void {
-    this.setUniform("canvasBackgroundColor", color);
-  }
-
```

Comment:
These were never actually being modified and aren't accessible on the UI. I decided to just remove the code paths outright.